pytorch_lightning/callbacks/pruning.py

```diff
@@ -128,6 +128,14 @@ def __init__(
 
             pruning_norm: if you are using ln_structured you need to specify norm.
 
+        Raises:
+            MisconfigurationException:
+                If ``parameter_names`` is neither ``"weight"`` nor ``"bias"``,
+                if the provided ``pruning_fn`` is not supported,
+                if ``pruning_dim`` is not provided when ``"unstructured"``,
+                if ``pruning_norm`` is not provided when ``"ln_structured"``,
+                if ``pruning_fn`` is neither ``str`` nor :class:`torch.nn.utils.prune.BasePruningMethod`, or
+                if ``amount`` is none of ``int``, ``float`` and ``Callable``.
```

Comment:
Is this too noisy?

![tmp](https://user-images.githubusercontent.com/20610905/107467303-1ae82a80-6ba9-11eb-8f2e-7d06414f8303.png)

Reply:
yes, can we make it as list?

Reply:
I actually tried to do so like the lines below, but the output doesn't change to the list view for some reason. Do you have any idea?

https://github.com/PyTorchLightning/pytorch-lightning/blob/84ae9f1ed2b9274783ff93f29e7164ae2e8e6751/pytorch_lightning/callbacks/pruning.py#L113-L117

![tmp](https://user-images.githubusercontent.com/20610905/107498511-3bc77480-6bd7-11eb-9d74-04cf3049a08e.png)

Reply:
You might want to hold on this until #5825 lands

Reply:
ok, lest postponed the Prune, but finish all others...

Reply:
It landed already, so no problem 🙂